index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable inclusive-language/use-inclusive-words */
```

Comment:
I'd prefer this be disabled just for the rule configuration section (as I'm guessing that's where there are errors) but we also want to get this feature out

Reply:
Fair point, I just figured all the lines in that section would need the exclusion which might make it less clear.